assessment/collectors/parser/response_parser.go

```diff
@@ -67,7 +87,7 @@ func ParseCodeImpact(codeImpactResponse map[string]any, filePath string) (*Snipp
 	}, nil
 }
 
-func ParseNonDaoFileChanges(fileAnalyzerResponse string, filePath string) ([]Snippet, []string, error) {
+func ParseNonDaoFileChanges(fileAnalyzerResponse string, filePath string, codeSnippetIndex *int) ([]Snippet, []string, error) {
```

Comment:
having the method change seems round about.

 Can you pass the index in ?

Reply:
Done.